client/scripts/views/pages/staking_calculator/chain_constants.ts

```diff
@@ -0,0 +1,64 @@
+import { ChainConstants } from '../../../models/ChainConstant'
+
+export const CHAIN_CONST: ChainConstants[] = [{
+    //Fetched from      : https://github.com/hicommonwealth/edgeware-node/blob/4da8fb14776a75d788a7a90c180887375bc9fca8/node/runtime/src/lib.rs#L251
+    //Fetced Date (UTC) : 2020-09-11T04:01:13Z
+    symbol: 'EDG',
+    const: {
+        minInflationRate: 0.025000,
+        maxInflationRate: 0.100000,
+        idealStakeRate: 0.800000,
```

Comment:
We should go ahead and change the idealStakeRate to 0.500000 (it will be updated in this next upgrade)